backups.txt

```diff
@@ -0,0 +1,19 @@
+// Parse JSON by default
+app.use(express.json());
+
```

Comment:
Don't push up this file, you can keep it locally if it helps, but have it gitignored